test/mock_windows.uts

```diff
@@ -167,21 +167,24 @@ test_read_routes6_windows()
 from scapy.config import conf
 assert conf.prog.powershell != None
 
-= Test all VBS fallback
+= Store powershell results
 import mock
 from scapy.config import conf
 
 ps_ip = get_ip_from_name(conf.iface.name)
+ps_ip
 ps_if_list = get_windows_if_list()
+ps_if_list
```

Comment:
(same)

Reply:
This is one of the expected values (ps means power shell)